Documentation/admission-controller-usage.md

```diff
@@ -0,0 +1,80 @@
+# Admission Controller
+
```

Comment:
After the header, how about a sentence or two that describes what an admission controller is and why we recommend they configure it.

Reply:
Added a short description.